vivisect/symboliks/analysis.py

```diff
@@ -471,7 +471,10 @@ def walkSymbolikPaths(self, fva, maxpath=1000):
         Begins first node by applying self.preeffects and self.preconstraints
         '''
 
-        graph = self.getSymbolikGraph(fva)
+        if symgraph == None:
+            graph = self.getSymbolikGraph(fva)
+        else:
+            graph = symgraph
 
```

Comment:
It's a total style-nit, but i'd rather we just rename "symgraph" to graph ( or all of "graph" to "symgraph" ) and eliminate the if/else block with a good ol'

<pre>
if graph == None:
    graph = self.getSymbolikGraph(fva)
</pre>